gh-pages-dist/assets/minimal-copy.css

```diff
@@ -0,0 +1,128 @@
+/* Opinionated reset */
```

Comment:
The file-name should be here also minimal-side right?

Reply:
This is an old file and going to be replaced with the next gh pages deployment